resources/resources.go

```diff
@@ -0,0 +1,194 @@
+package resources
+
+import (
+	"context"
+
+	"golang.org/x/xerrors"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	v1 "k8s.io/client-go/applyconfigurations/core/v1"
+	confstoragev1 "k8s.io/client-go/applyconfigurations/storage/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	v1beta2config "k8s.io/kube-scheduler/config/v1beta2"
+)
+
+type Service struct {
+	client              clientset.Interface
+	podService          PodService
+	nodeService         NodeService
+	pvService           PersistentVolumeService
+	pvcService          PersistentVolumeClaimService
+	storageClassService StorageClassService
+	schedulerService    SchedulerService
+}
+
+// Resources  will used to compile all the resources for export
+type Resources struct {
+	PodList          *[]corev1.Pod                             `json:"podList"`
```

Comment:
Please rename all fields from `HogeList` to `Hoges`. (For example: `PodList` → `Pods`)

Reply:
And please change type from `*[]corev1.Pod` to `[]corev1.Pod`. (other fields as well)
I don't see any reasons to use a pointer of slices.